debian/gen-flat-repo-fixtures.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eux
+
+SRCDIR=$(readlink -f "$(dirname "$0")")
+OUTPUTDIR=$(realpath "${SRCDIR}/../$1")
+
+mkdir "${OUTPUTDIR}"
+
+cd "$OUTPUTDIR"
+(
+  for CTL in "${SRCDIR}"/asgard/*.ctl
+  do
+    equivs-build --arch ppc64 "${CTL}"
+  done
+)
+
+cd ..
+dpkg-scanpackages debian-flat/ | tee debian-flat/Packages | gzip > debian-flat/Packages.gz
```

Comment:
Right now we are building a flat repo with the `fixtures` folder as its repository root, plus a distribution of `debian-flat/`.
I think we could shift things around ever so slightly to get two fixtures for the price of one:
Suppose we made `fixtures/debian-flat/` the repository root, and prepared two sets of metadata files. One in the repository root for a distribution of `/` (I think this is the most common case of flat repos in the wild), and another one in some twice nested subfolder (e.g.: `fixtrues/debian-flat/nest/fjalar/`) for a distribution of `nest/fjalar/`. (Twice nested because problems with the code often only start appearing at twice nested and we are trying to test the code).

Reply:
Done. All `*.deb` files and `Release` and `Packages` are created in `fixtures/debian-flat` and in `fixtures-debian-flat/nest/fjalar`.